src/pages/profile/OtherProfile.tsx

```diff
@@ -22,37 +31,62 @@ export const OtherProfile: FC = () => {
         const foundUser = response?.user as unknown as User;
         const newPosts = response?.posts as unknown as Post[];
         if (foundUser && newPosts) {
-          setUser(foundUser);
+          setOtherUser(foundUser);
           setPosts(newPosts);
         }
       })();
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [user]);
+  }, [otherUser]);
+
+  const toggleFollow = async () => {
+    if (otherUser) {
+      const response = await operation({
+        method: "put",
+        url: `/follow`,
+        data: { followId: otherUser._id },
+      });
+      console.log("response", response);
+
+      if (response) {
```

Comment:
Suggestion: check for console.log here too

Reply:
done